### PR TITLE
executor: Support new table scan executor

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @hanfei19910905 @coocood @shenli

**Comment by hanfei19910905** [comment]
LGTM

File: executor/new_builder.go [hunk 1]
```diff
@@ -155,21 +157,35 @@ func (b *executorBuilder) buildNewTableScan(v *plan.NewTableScan, s *plan.Select
...
+		t:          table,
+		asName:     v.TableAsName,
+		ctx:        b.ctx,
+		columns:    v.Columns,
+		schema:     v.GetSchema(),
+		seekHandle: math.MinInt64,
+		ranges:     v.Ranges,
+	}
+	if v.Desc {
+		b.err = errors.New("Not implement yet.")
```
Comment (coocood) [inline executor/new_builder.go]: We don't need to return error, there is ReverseExec executor can handle this.

**Comment by coocood** [comment]
LGTM

**Pull request merged**